src/components/book/Book.js

```diff
@@ -1,13 +1,26 @@
 import Author from '../author/Author';
 import './Book.css';
+import { Link } from 'react-router-dom';
 
 function Book({
-  book: { title, total_pages: totalPages, rating, isbn, published_date: publishedDate, authors },
+  book: {
+    title,
+    total_pages: totalPages,
+    rating,
+    isbn,
+    published_date: publishedDate,
+    authors,
+    book_id,
+  },
   showDetail = false,
 }) {
   return (
     <article className="book">
-      <h3>{title}</h3>
+      {!showDetail && (
+        <Link to={`/books/${book_id}`}>
```

Comment:
This logic is so close but your title isn't displaying when you are on the detail page!